Update github link and description for Madoko.

CommunicationAnimale opened the pull request:
Point Madoko's GitHub link to the actual repo, and update the description to match that of the reference manual.

erquhart commented:
@CommunicationAnimale thanks for the update!

Some of this is more than what's really needed for staticgen.org - the existing description isn't contrived, it's pulled from the website. Can you trim this down to just update the url and repo?

CommunicationAnimale commented:
Indeed I didn't realize the text I copy-pasted was so technical. I updated the commit to keep the previous text. I still kept the first paragraph of the reference manual as I think it nicely summarizes Madoko's ambitions. I also updated the link to the reference manual and fixed minor typos.

CommunicationAnimale commented:
Rebased master.
